src/relay/pass/quantize.cc

```diff
@@ -216,15 +215,21 @@ Expr QuantizeRealize(const Call& ref_call,
     }
 
     float shift_nbit = std::log2(odom_scale_imm / idom_scale_imm);
-    CHECK_GT(shift_nbit, 0);
+    CHECK_NE(shift_nbit, 0);
```

Comment:
If negative happens with KL calibration, we should also handle `shift_nbit == 0` too

Reply:
@ZihengJiang `shift_nbit==0` is handled in `idom_scale_imm == odom_scale_imm` check